### PR TITLE
OMNIBUS F3 initial support (OSD-less)

### PR DESCRIPTION
**Pull request by jflyper**
As per #568 

See Boards - OMNIBUS for configuration details.

### CODE REVIEW

**Comment by digitalentity** [comment]
@jflyper thanks! I'll review and merge once I have the time.

**Comment by jflyper** [comment]
Ah, sorry, I specified the wrong branch; I read the 1.2.1 PR after this PR was created and came to knew the PR policy.

File: src/main/target/OMNIBUS/target.c [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+const uint16_t multiPPM[] = {
+    PWM1  | (MAP_TO_PPM_INPUT    << 8), // PPM input
+
+    PWM2  | (MAP_TO_MOTOR_OUTPUT << 8),
+    PWM3  | (MAP_TO_MOTOR_OUTPUT << 8),
+    PWM4  | (MAP_TO_MOTOR_OUTPUT << 8),
+    PWM5  | (MAP_TO_MOTOR_OUTPUT << 8),
+    PWM6  | (MAP_TO_MOTOR_OUTPUT << 8),
+    PWM7  | (MAP_TO_MOTOR_OUTPUT << 8),
+#if 0
```
Comment (martinbudden) [inline src/main/target/OMNIBUS/target.c]: Can you delete code rather than use `#if`

**Comment by digitalentity** [comment]
Seems to be working,

**Comment by digitalentity** [comment]
Guys, any objections against merging this into upcoming 1.3?

**Comment by martinbudden** [comment]
Go ahead, no objections.

**Comment by jflyper** [comment]
@digitalentity :cool: Thanks for testing.

**Comment by digitalentity** [comment]
@jflyper I didn't actually flight-test it, I merely verified that it's working (at least to certain level). I also trust Betaflight's devs :smile:

**Pull request merged**